man/man5/gitsh_completions.5.in

```diff
@@ -267,6 +305,18 @@ and
 wouldn't.
 .El
 .
+.Sh MODIFIERS
+The meaning of variables can be modified by appending a suffix.
```

Comment:
This sentence could use rewording for the new syntax.

Reply:
Good catch, thanks!